dev-qt/qt5/qt5-5.6.1.recipe

```diff
@@ -9,7 +9,7 @@ LICENSE="GNU LGPL v2"
 
 SOURCE_URI="https://download.qt.io/official_releases/qt/5.6/$portVersion/single/qt-everywhere-opensource-src-$portVersion.tar.xz"
```

Comment:
I forgot to explain why we try to avoid xz archives when possible: haikuporter is not (yet) able to extract *.tar.xz archives with on-the-fly decompression. It needs to unxz them first, and then it calls tar to extract the uncompressed tarball (before deleting it.) So for very big archives this can take a while...
And the other reason why we try to avoid xz archives is that xz is not available for non hybrid x86_gcc2.